### DISCUSSION TITLE
The Huge Walls For Who Wants To Start Game

### DISCUSSION THREAD

**Discussion by Paiden7** [sunflower-land/sunflower-land#2655] [Ideas]
Hi SFL community. I want you to tell something about a huge wall for who want to start this game. First of all i started 2 weeks ago. Game has nice graphic and consept for me. Be a farmer, earn money. But when i started there is no step by step video or guide for me. There was just people who has full expansion and all nfts says "You can buy seed from here. Chop trees. You need stone for iron pickaxe." There was nothing about strategy, what nfts should i need to buy for early game, mid game, late game. I found all of them myself with asking people, reading all whitepaper, watching streams. People who wants to start game dont wanna effort this kind of things. They just want to see a video about it. I couldnt find a new video to help me. They all was 2 to 3 months ago. 2 or 3 month is much for this kind of game. Everything could get changed with just 1  update. New players need a video guide who starts from 0, tells the strategies. Example get more crops, more wood, more rocks, more sfls, more gold. Game could have tons of strategies because game is around nfts. Think about someone who has 15 dolars. Do you advise him to buy woody for wood strategie? Ofc no he will do others. After time he got the point of the game and go from his own way like us. Luckly i was in crypto since 2018. I got the main idea fastest i could but not everyone is in crypto. And this part we will go for second huge wall for Who wants to start game. I mean before start. Most people want to just click the button plant, join events, have fun and earn money. Not they are here for creating wallet accound, add polygon, send money on polygon chain, open game on wallet browser. They dont even know what is chain is. They are here for fun or money. They dont want to effort all of this. In my opinion we need a mobile app with in game basic wallet for them. Everyone has phones with app stores but not everyone has metamask to playing game from in it. They should click,download and play,need basic setup. Thx.

**Comment by Jericho-rpc** [comment]
Well, at least the player should be given money and fun. Currently, neither is terribly bad. This is very confusing and illogical. The game side needs to help  players determine how to act and what to move forward
